hw2/ViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ViewController.swift
+//  hw2
+//
+//  Created by Tina Jureško on 14.03.2025..
+//
+
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class ViewController: UIViewController {
+
+    private let leagueView = LeagueView()
+    //private let matchView = MatchView()
+    
+    private let matchesStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        view.addSubview(leagueView)
+        leagueView.translatesAutoresizingMaskIntoConstraints = false
+        leagueView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(20)
+            make.leading.equalTo(view.safeAreaLayoutGuide.snp.leading).offset(16)
+            make.trailing.equalTo(view.safeAreaLayoutGuide.snp.trailing).offset(-16)
+            make.height.equalTo(56)
+        }
```

Comment:
Možeš korisititi shorthand argument name to ti onda izgleda ovako

homeTeamLogo.snp.makeConstraints {
$0.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(20)
$0.leading.equalTo(view.safeAreaLayoutGuide.snp.leading).offset(16)
...
}

To nam swift pruža da ne moramo explicitno imenovati argument, jer u ovom slučaju znamo o čemu se radi.